lib/configuration/variables/sources/instance-dependent/get-ssm.js

```diff
@@ -21,6 +21,8 @@ module.exports = (serverlessInstance) => {
       });
       const region = !params || !params[0] || params[0] === 'raw' ? undefined : params[0];
       const shouldReturnRawValue = params && (params[0] === 'raw' || params[1] === 'raw');
+      const shouldSkipDecryption =
+        params && (params[0] === 'noDecrypt' || params[1] === 'noDecrypt');
```

Comment:
To secure proper resolution of all params, I'll probably go that route:

```javascript
const shouldReturnRawValue = params.includes('raw');
const shouldSkipDecryption = params.includes('noDecrypt');
_.pull(params, ['raw', 'noDecrypt']);
const region = params[0];
```

Reply:
wow, awesome suggestion